gui/slick/js/new/config_subtitles.js

```diff
@@ -0,0 +1,13 @@
+$(document).ready(function() {
```

Comment:
For now it's just what was in the mako files, I'm slowly removing this. I'll redo the name and stuff for this PR in a bit.